### ISSUE TITLE
Expose underlying OperationsClient

### ISSUE THREAD

**Issue by nnegrey** [googleapis/gapic-generator-python#274]
For Long Running Operations, it'd be nice to be able to get direct access to the operations client used by some of the libraries. 

For example in AutoML, to get access to the operations client, you have to access the internal method:
`client.transport._operations_client`

```
client = automl.AutoMlClient()
response = client.transport._operations_client.list_operations(...)
```

Other languages provide a method to get access to the operations client. 
Java:
`client.getOperationsClient()`
Node.js
`client.operationsClient`

**Comment by software-dov** [comment]
Why? What use case does this address?

**Comment by nnegrey** [comment]
Product teams want docs / samples that demonstrate how to get the status of LROs, it also has come up in some customer issues.

Folks get back an Operation Id for a long running operation may not want to wait for the operation but check back later.

**Comment by software-dov** [comment]
I think we need a clearer idea of the problem we're trying to solve before we start making changes.
Getting the status of an LRO is easy: the api_core wrapper type exposes methods and properties that let us query whether it's done, whether it succeeded, what the error (if any) was, and what the result is (if any). See [operation.py](https://github.com/googleapis/google-cloud-python/blob/master/api_core/google/api_core/operation.py) and [polling.py](https://github.com/googleapis/google-cloud-python/blob/master/api_core/google/api_core/future/polling.py) for details. It could be that documentation of this capability is a lacuna in the generated surface, but it exists.

It sounds like what our users may really want is resumable LROs, or transmission of an operation across a process boundary. I don't think the GAPIC surface currently exposes a mechanism for sending an operation ID to a fresh Operations client and getting back an Operation object. If that is what our users want, then that functionality definitely belongs in the generator and not in google-cloud-python.

**Comment by nnegrey** [comment]
I just want a method that exposes the existing operations client the same way other languages do without having to use internal methods.

As from the java docs:
```
Returns the OperationsClient that can be used to query the status of a long-running operation
returned by another API method call.
``` 

[Java - AutoML](https://github.com/googleapis/java-automl/blob/master/google-cloud-automl/src/main/java/com/google/cloud/automl/v1beta1/AutoMlClient.java#L175-L183)
[Java - Speech](https://github.com/googleapis/java-speech/blob/master/google-cloud-speech/src/main/java/com/google/cloud/speech/v1/SpeechClient.java#L164-L172)
These both return the [google.longrunning.OperationsClient.java](https://github.com/googleapis/gax-java/blob/master/gax-grpc/src/main/java/com/google/longrunning/OperationsClient.java) that my existing client is already using.

[PHP](https://github.com/googleapis/google-cloud-php/blob/master/Speech/src/V1/Gapic/SpeechGapicClient.php#L128-L137)
[Node.js](https://github.com/googleapis/nodejs-automl/blob/master/src/v1/auto_ml_client.js#L183-L189)

Not looking for additional functionality, just a `get_operations_client()` method.

**Comment by software-dov** [comment]
Duplicate of #640
